examples/connectivity/plot_adhd_covariance.py

```diff
@@ -95,13 +98,24 @@ def plot_matrices(cov, prec, title):
 
 # Displaying results ##########################################################
 print("-- Displaying results")
-title = ("Subject {0:d} GroupSparseCovariance "
-         "$\\alpha={1:.2e}$").format(plotted_subject, gsc.alpha_)
+atlas_imgs = image.iter_img(msdl_atlas_dataset.maps)
+atlas_region_coords = np.array(
+    [plotting.find_xyz_cut_coords(img) for img in atlas_imgs])
+
+title = "Subject {0:d} GroupSparseCovariance $\\alpha={1:.2e}$".format(
```

Comment:
Feel free to create an issue about that. If we had to chose one I'd be personally in favour of using `.format` rather than `%`.